docs/src/manual/ansatz/mps.md

```diff
@@ -1,8 +1,8 @@
 # Matrix Product States (MPS)
 
-Matrix Product States ([`MPS`](@ref)) are a Quantum Tensor Network ansatz whose tensors are laid out in a 1D chain.
+Matrix Product States ([`MPS`](@ref)) are a Quantum Tensor Network [`Ansatz`](@ref) whose tensors are laid out in a 1D chain.
 Due to this, these networks are also known as _Tensor Trains_ in other scientific fields.
-Depending on the boundary conditions, the chains can be open or closed (i.e. periodic boundary conditions), currently
+Depending on the boundary conditions, these chains can be open or closed (i.e. periodic boundary conditions). Currently
 only `Open` boundary conditions are supported in `Tenet`.
```

Comment:
Markdown will put any phrase that is in the next line, and that it's not a blank line, just in continuation. So you can split into multiple lines if you want without problem.
```suggestion
Depending on the boundary conditions, these chains can be open or closed (i.e. periodic boundary conditions).
Currently only `Open` boundary conditions are supported in `Tenet`.
```